container/features/src/main/resources/features.xml

```diff
@@ -1839,4 +1839,12 @@
     <feature name="opennms-core-ipc-twin-shell" description="OpenNMS :: Core :: IPC :: Twin :: Shell" version="${project.version}">
         <bundle>mvn:org.opennms.core.ipc.twin/org.opennms.core.ipc.twin.shell/${project.version}</bundle>
     </feature>
+
+    <feature name="opennms-deviceconfig" description="OpenNMS :: Features :: Device Config">
```

Comment:
Rename it as `opennms-deviceconfig-tftp` ?

Reply:
I also wondered how fine granular we should be. I thought that a single feature would be enough.

If we decide to use a single feature (for all client related functionality) then maybe we should rename it into `opennms-deviceconfig-client`. (The corresponding `opennms-deviceconfig-server` feature would contain the `pollerd` related things.)

If we decide to use several features then the `shell` submodule (that contains a karaf command for exploring the tftp server) should also be renamed (maybe into `tftp-shell`).